cookiecutter.json

```diff
@@ -1,8 +1,9 @@
 {
     "full_name": "Name or Organization",
+    "project_name": "MyProject",
```

Comment:
Let's not have caps/camelcase, so "myproject", but we could also use "diffpy.myproject" to remind people that they can use "a.b.project" pattern?

Reply:
or "my-project"